src/Meshes/abstractMesh.ts

```diff
@@ -1371,6 +1377,11 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
                         }
                     }
                 }
+
+                if (this._positions) {
+                    const index = vertexCount * 3;
+                    this._positions[vertexCount].copyFromFloats(data[index], data[index + 1], data[index + 2]);
```

Comment:
Shouldn't we take the `if` branch only if we modified `data[vertexCount]` starting line 1373? Else, it will overwrite the changes that the skeleton code could have made.